Userland/Libraries/LibWeb/HTML/FormControlInfrastructure.cpp

```diff
@@ -175,4 +176,91 @@ WebIDL::ExceptionOr<Optional<Vector<XHR::FormDataEntry>>> construct_entry_list(J
     return entry_list;
 }
 
+// https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#multipart-form-data
+ErrorOr<SerializedFormData> serialize_to_multipart_form_data(Vector<XHR::FormDataEntry> const& entry_list)
+{
+    auto normalize_line_breaks = [](StringView value) -> ErrorOr<String> {
+        StringBuilder builder;
+        GenericLexer lexer { value };
+        TRY(builder.try_append(lexer.consume_until(is_any_of("\r\n"sv))));
```

Comment:
Can we move this into the loop and then remove the last line from it? Should be functionally equivalent